### ISSUE TITLE
country.is restricted by usage

### ISSUE THREAD

**Issue by Fitrixx** [hakanensari/country#33]
Thank you so much for all ur work <3 much appreciate it

so sometimes i get response 429 too much request o i get over this and can i ask how much can one ip request ?

i'm using it with CURL and PHP <3

**Comment by AbhishekTaparia** [comment]
Same question. Is there any throttling for this? How many request per sec or min..?

**Comment by hakanensari** [comment]
Yes, I have rate limiting enabled—100 requests per 10 seconds, per IP. The 422 error response will now clarify this.

For bulk queries, consider downloading the database and querying it locally.
